src/draw/lv_draw_private.h

```diff
@@ -54,12 +54,18 @@ struct _lv_draw_task_t {
      * Therefore during drawing the layer's clip area shouldn't be used as it might be already changed for other draw tasks.
      */
     lv_area_t clip_area;
+    lv_layer_t * target_layer;
 
 #if LV_DRAW_TRANSFORM_USE_MATRIX
     /** Transform matrix to be applied when rendering the layer */
     lv_matrix_t matrix;
 #endif
 
+#if LV_USE_PARALLEL_DRAW_DEBUG
```

Comment:
This draw_unit member can be resident, and there is no need to judge `LV_USE_PARALLEL_DRAW_DEBUG`. Use `t->draw_unit` to access the context.